net/tailscale/test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+if command -v tailscale; then
+    tailscale version | grep "$2"
```

Comment:
If this fails and the tailscaled test below succeeds, then the test script will exit with a success exit value. I suggest either exiting immediately if this test fails, or adding a line at the end of the script that evaluates the exit value of both tests.

Reply:
fixed